### DISCUSSION TITLE
support data upload to pvoutput.org

### DISCUSSION THREAD

**Discussion by Kugeleis** [arendst/Tasmota#12315] [Q&A] [answered]
I am using a plug with tasmota to monitor my small solar power generator. I would like to upload the measured values from tasmota to the web service like so

- `curl -d "data=20210830,15000" -H "X-Pvoutput-Apikey: Your-API-Key" -H "X-Pvoutput-SystemId: Your-System-Id" https://pvoutput.org/service/r2/addoutput.jsp`
- `https://pvoutput.org/service/r2/addoutput.jsp?key=Your-API-Key&amp;sid=Your-System-Id&amp;d=20100830&amp;g=12000`
I see the websend feature does not support https. Could this be included?

**Comment by sfromis** [comment, marked as answer]
WebSend support for https is not likely, unless someone creates the feature in a well-working PR. What I'd do would be to have a small gateway in Node-Red (or similar tool), to bridge the gap.

**Comment by Kugeleis** [comment]
I wonder if I could include a user defined https certifiate and endpoint like [it was done with telegram](https://github.com/arendst/Tasmota/issues/8619) in my compile.
I just don't see how to do this.
